### PR TITLE
Update banner colors

### PR DESCRIPTION
**Pull request by hursey013**
Fixes https://github.com/18F/samhsa-prototype/issues/79

Updates beta banner to softer yellow and helpline to teal.

Screen Shot 2019-07-16 at 3 17 11 PM

### CODE REVIEW

**Comment by allisonnorman** [comment]
It looks greener than the InVision version, which has more of a blue tone, which I prefer. @nickttng - Are you sure those are the right hex codes / what do you think?

**Comment by nickttng** [comment]
The Tailwind default teal is a bit more green than the proposed custom color; however, I think we can override the default teal with proposed teal with new hex code(s). I can check Tailwind doc on how it could be done.

**Comment by hursey013** [comment]
@nickttng Sorry I missed that there was a custom hex code in the issue, I can update it with that.

**Comment by nickttng** [comment]
The new teal palette looks good.

File: src/tailwind.js [hunk 1]
```diff
@@ -11,6 +11,19 @@ module.exports = {
...
+        teal: {
+          '100': '#D8EBF2',
+          '200': '#ADE3F7',
+          '300': '#73CEF0',
+          '400': '#45BAE6',
+          '500': '#2C98C0',
+          '600': '#2285A9',
+          '700': '#196986',
+          '800': '#0F4756',
+          '900': '#033340'
```
[the remark this answers is not in the document]
Comment (hursey013) [reply src/tailwind.js]: Trailing commas are removed as per our code formatting styles: https://github.com/18F/samhsa-prototype/blob/master/.prettierrc.json#L4

**Pull request merged**